lib/utils/firebase.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:async';
+
+import 'package:firebase_analytics/firebase_analytics.dart';
+import 'package:firebase_analytics/observer.dart';
+import 'package:firebase_crashlytics/firebase_crashlytics.dart';
+import 'package:flutter/foundation.dart';
+
+/// Analytics - Initialises and provides helper methods for Firebase Google Analytics
+class Analytics {
+  static FirebaseAnalytics analytics = FirebaseAnalytics();
+  static FirebaseAnalyticsObserver observer =
+      FirebaseAnalyticsObserver(analytics: analytics);
+
+  Future<void> analyticsScreen(String screenName) async {
```

Comment:
We can make it `static` and then we don't need to create a new instance of this class every time.

Reply:
oh yeah should be ... oops 👍@maks flagged it too.

Reply:
done